src/components/Tile/Tile.styles.ts

```diff
@@ -0,0 +1,41 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { TileAspectRatio } from './Tile.types';
+import { memoizeFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { mergeStyleSets, ITheme } from 'office-ui-fabric-react/lib/Styling';
+
+export interface TileClassNameProps {
+  aspectRatio: TileAspectRatio;
+  theme: ITheme;
+}
+
+const mapAspectRatioToPercent: Record<TileAspectRatio, string> = {
+  [TileAspectRatio.FourThree]: '75%',
+  [TileAspectRatio.SixteenNine]: '56.25%',
+  [TileAspectRatio.TwentyOneNine]: '42.85%',
+};
+
+export const getClassNames = memoizeFunction((classNameProps: TileClassNameProps) => {
+  const { aspectRatio, theme } = classNameProps;
+
+  return mergeStyleSets({
+    root: {
+      backgroundColor: theme.semanticColors.bodyDivider,
+      border: `1px solid ${theme.semanticColors.bodyDivider}`,
```

Comment:
Would we place anything besides an image as Tile content? Since this is a layout component, I would consider whether we want to enforce a border here or if that would be up to the application to place on the content.

Also note that the previous implementation used an inset 1px shadow overlay to darken the image's outer pixels rather than pushing the image in and placing the border outside. Not sure what the Designs call for, but looking at the "with text" example you probably want to do inset shadow instead of border.

Reply:
For now, the only designs we have all have a border around the content, regardless of whether they're images or text.

True about the inset shadow, I'll fix that.